src/UI/README.md

```diff
@@ -391,6 +391,118 @@ If you would like to implement a new component to the framework you should perfo
 
 TODO: write me
 
+## Abstraction of Javascript in the Framework
+
+_Note: The concept described in this section is not yet fully finalized._
+
+The functionality of some components relies on javascript. As an example, a modal is showed and closed by 
+clicking on some triggerer component. As a user of the framework, you should not worry about writing the javascript
+logic for this kind of interactions.
+
+### About Triggerables, Triggerer and Signals
+
+Before describing the concept in more detail, you should be aware of the following definitions:
+* **Signal** A signal describes a javascript action of a component which can be triggered by another component of the
+framework.
```

Comment:
I understand the argument for leaving out the term javascript itself. but for many developers it will be a helpful detail to get the dry facts categorized in mind while studying i think. imho a certain level in the middle of concrete and abstract should be chosen. I like current state :-)